cmd/server.go

```diff
@@ -74,7 +74,7 @@ var serverCmd = &cobra.Command{
 		*/
```

Comment:
Not a good place to put this comment, but can you add some context to the commit message. Something along the lines of high level of how the tests are supposed to work.

Reply:
tests? sorry, not following. I can put this comment showing use of a different persistence layer in a doc tho.

Reply:
fyi, removed comments